### ISSUE TITLE
Add a step for installation

### ISSUE THREAD

**Issue by lipingtababa** [huaweicloud/terraform-provider-huaweicloudstack#1]
before 
```go get github.com/terraform-providers/terraform-provider-huaweicloudstack```
users have to enable git prompt first as below

```
export GIT_TERMINAL_PROMPT=1
```

otherwise they would come through an error like 
```
fatal: could not read Username for 'https://github.com': terminal prompts disabled
```

**Comment by niuzhenguo** [comment]
@lipingtababa Seems only private repo will cause such error, and it's OK on my ENV.

**Comment by niuzhenguo** [comment]
closed with #2
